extensions/sql-database-projects/src/controllers/mainController.ts

```diff
@@ -189,13 +186,6 @@ export default class MainController implements vscode.Disposable {
 		}
 	}
 
-	private registerProjectProvider(): void {
```

Comment:
will we add it back pot sept? or add itsome other way to avoid hard dependency?

Reply:
I'll port this to release